apps/parsley/src/components/LogRow/SectionHeader/SubsectionControls.tsx

```diff
@@ -0,0 +1,45 @@
+import Button from "@leafygreen-ui/button";
+import { useLogContext } from "context/LogContext";
+
+const SubsectionControls: React.FC<{ functionID: string }> = ({
+  functionID,
+}) => {
+  const { sectioning } = useLogContext();
+  const { sectionState, toggleAllCommandsInFunction } = sectioning;
+  const showExpandButton =
+    sectionState &&
+    (Object.values(sectionState[functionID].commands).some(
+      (commandID) => !commandID.isOpen,
+    ) ||
+      !sectionState[functionID].isOpen);
+  const showCloseButton =
```

Comment:
Would it make sense to memoize these values?

Reply:
I don't think so since all callbacks from `useSections` are already memoized.